<ui version="4.0" >
 <class>NetWorkSetting</class>
 <widget class="QDialog" name="NetWorkSetting" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CoolView - 网络设置</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="wengoPhoneBuildIdStringLabel" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>301</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton" >
       <property name="text" >
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>112</height>
      </size>
     </property>
     <property name="title" >
      <string>设置</string>
     </property>
     <widget class="QLabel" name="label" >
      <property name="geometry" >
       <rect>
        <x>30</x>
        <y>30</y>
        <width>54</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text" >
       <string>IP:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2" >
      <property name="geometry" >
       <rect>
        <x>30</x>
        <y>60</y>
        <width>54</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text" >
       <string>地址类型:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3" >
      <property name="geometry" >
       <rect>
        <x>30</x>
        <y>90</y>
        <width>54</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text" >
       <string>支持组播:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="IPComboBox" >
      <property name="geometry" >
       <rect>
        <x>90</x>
        <y>20</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="addrTypeComboBox" >
      <property name="geometry" >
       <rect>
        <x>90</x>
        <y>50</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text" >
        <string>IPv4</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>IPv6</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="isMulticastComboBox" >
      <property name="geometry" >
       <rect>
        <x>90</x>
        <y>80</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text" >
        <string>是</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>否</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../resources/wengophone.qrc" />
  <include location="../../../../resources/wengophone.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>NetWorkSetting</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>453</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel" >
     <x>212</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
